plugins/templates/getTemplateById.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const boom = require('@hapi/boom');
+const joi = require('joi');
+const schema = require('screwdriver-data-schema');
+const getSchema = schema.models.template.get;
+const idSchema = schema.models.template.extract('id');
+
+module.exports = () => ({
+    method: 'GET',
+    path: '/templates/{id}',
+    options: {
+        description: 'Get a single template',
+        notes: 'Returns a template record',
+        tags: ['api', 'templates'],
+        auth: {
+            strategies: ['token'],
+            scope: ['user', 'build']
+        },
+
+        handler: async (request, h) => {
+            const { templateFactory  } = request.server.app;
+
+            return templateFactory
+                .get(request.params.id)
```

Comment:
According to https://github.com/screwdriver-cd/models/blob/4ffe0884a72488b743d20dbdb83d70eb0a933857/lib/baseFactory.js#L71,  I believe this

```
.get(request.params.id)
```

should be an object with id

```
.get({ id: request.params.id })
```

cc @tkyi

Reply:
Tiff told me that, bothy should be okay. So no changes needed.